darwin/dataset/remote_dataset.py

```diff
@@ -60,8 +61,28 @@ def _add_files_to_dataset(self, filenames: List, fps: int):
             self._client.put(f"/dataset_videos/{metadata['id']}/confirm_upload", payload={})
             yield
 
-    def pull(self):
-        """Downloads a rermote project (images and annotations) in the projects directory. """
+    @staticmethod
+    def _f(x):
+        """Support function for pool.map() in pull()"""
+        x()
+
+    def pull(self, blocking:Optional[bool] = True, multi_threaded : Optional[bool] = True):
```

Comment:
Here, a formatter would probably complain
```suggestion
    def pull(self, blocking: Optional[bool] = True, multi_threaded: Optional[bool] = True):
```